.gitignore

```diff
@@ -1,3 +1,4 @@
 Manifest
 pkg
 output
+.idea
```

Comment:
Yeah, but I actually do commit some files from Idea to some of my repositories so I don't keep it in the global gitignore.